Functions/Client.cs

```diff
@@ -113,26 +126,34 @@ private async Task<HttpResponseMessage> HandleRequest(string url, string? token
 
             options.Headers["X-Client-Info"] = Util.GetAssemblyVersion(typeof(Client));
 
+            if (options.FunctionRegion != FunctionRegion.Any)
+            {
+                options.Headers["x-region"] = options.FunctionRegion.ToString();
+            }
+

```

Comment:
The client can also be initialized with a region, https://github.com/supabase/functions-js/blob/main/src/FunctionsClient.ts#L23

Then on invoke, should get region first from `options`, otherwise from the client itself https://github.com/supabase/functions-js/blob/main/src/FunctionsClient.ts#L58